### PR TITLE
Create Apple-Inc webapp

### PR DESCRIPTION
**Pull request by skabeo**
In this pull request

- Created a web app to view the financial statements of Apple Inc for the year ended 2022.
- Used redux to store the statements data.
- Displayed a preview of each statement on the Homepage.
- On click of a statement, users are sent to a view the detailed statement.
- Users can filter the list by typing into the search field.
- The message `No search results` is displayed when the user types in something in the search field and doesn't appear in the list of items displayed.
- When the app is mounted there is a nice loading spinner loader displayed when the actions to retrieve the data are in the pending state.
- Finally tested each components with `jest library`.

### CODE REVIEW

File: src/components/IncomeStatement.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import Back from './Back';
+import styles from '../styles/Details.module.scss';
+import Filter from './Filter';
+
+const IncomeStatement = () => {
```
Comment (julie-ify) [inline src/components/IncomeStatement.js:7]: - [ ] I see you have search functionality in all the components. You are free to implement this search in all components as your own added preference however you should have a search feature in the Home component as stated in the requirements.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/" element={<Homepage />} />
+        <Route path="/income-statement" element={<IncomeStatement />} />
+        <Route path="/balance-sheet" element={<BalanceSheet />} />
+        <Route path="/cash-flow" element={<Cashflow />} />
+        <Route path="/equity" element={<Equity />} />
+      </Routes>
```
Comment (julie-ify) [inline src/App.js]: - [ ] We should basically only have two routes, the home route, and the details page route, or at most three. Each item on the home page should have its own unique route within the details page. Right now your items route is something like this:

  ```
  localhost:3000/equity
  ```

  Whereas you should have something like this:

  ```
  localhost:3000/details/equity
  ```

  You can consider using a slug for this feature. You can use the param from the route to display data based on that. See this example: https://reactrouter.com/en/main/hooks/use-params

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @skabeo,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @skabeo 👋 

## Things to Appreciate 
- [ ] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [ ] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**